src/modules/expression-evaluator/expression-evaluate-gui/Z_To-Do.todo

```diff
@@ -0,0 +1,18 @@
+Important:
```

Comment:
Oh no! I think all the `expression-evaluator-gui` changes end-up been added to this PR. Can you remove those please?

Reply:
It's merged with `master` and `#77-sequential-actions` now, so they no longer show up in the diff.

Reply:
Nice! I think Andrei had a look too. That's good.